src/main/scala/medeia/generic/CoproductDerivationOptions.scala

```diff
@@ -0,0 +1,8 @@
+package medeia.generic
+
+case class CoproductDerivationOptions[A](
+    typeNameTransformation: PartialFunction[String, String] = PartialFunction.empty,
+    typeNameKey: String = "type"
```

Comment:
WDYT about `typeTagAttr` or `typeTag` or `typeTagName` or something like that?

Reply:
done